src/Symfony/Component/Notifier/Bridge/Bandwidth/BandwidthTransport.php

```diff
@@ -31,7 +31,8 @@ final class BandwidthTransport extends AbstractTransport
 
     public function __construct(
         private readonly string $username,
-        #[\SensitiveParameter] private readonly string $password,
+        #[\SensitiveParameter]
+        private readonly string $password,
```

Comment:
any opinion on this change @symfony/mergers?
which one is the most readable to you?
on my side, I don't mind long lines.

Reply:
Im happy with the two lines. I think it is more readable. Especially if you have very long attributes.

Reply:
I like the one line more

Reply:
+1 for long one-liners. 2 lines break my brain when reading the property list

Reply:
Oh.. I dont want to be responsible for breaking Robin's brain.

Reply:
So shall I revert all bridges to one line?

In all docs, and especially if we use attributes with parameters, we tend to use two lines for readability, because
```php
#[MapRequestPaylod(foo: bar, baz: baz)] private readonly Class $foo
```

is quite noisy.

Please advice and I will take the wanted direction across all bridges

Reply:
I really depends on the length of the attribute part.
When a line-split feels needed, I think we should require adding blank lines before and after the corresponding argument declaration.
In this case, the oneliner is better to me, so yes, please update all lines found in this PR and other similar PRs, if any :pray:

Reply:
Will do

Reply:
In the docs, we follow Symfony coding standards except from the line length. This is in part also because code blocks in the documentation are at most 726px wide, we don't want to overwhelm readers with too much characters and code examples are often very short.

In Symfony code, we often favor longer lines. This is in part because editors are often fullscreen on a wide screen and having more "real" lines vertically allows you to see code more in context (at least, that's how I experience it). This is also why we e.g. always one-line exceptions.

Reply:
Done